### DISCUSSION TITLE
Disk export feature no longer works

### DISCUSSION THREAD

**Discussion by ItsIgnacioPortal** [projectdiscovery/nuclei#802] [Q&A] [answered]
#195 discusses the "-disk-export" feature. I'm looking for a way to "store complete HTTP request/response in a single file as a poc", but it seems like this feature was never added to the -help menu, nor the documentation. Do we know on which version this feature was removed/broken? As of v2.3.8 Nuclei reports:

```
nuclei -v -disk-export /home/kali/tmp/nuclei/nucleilog -t $HOMEDIR/nuclei-templates/vulnerabilities/ example.com

flag provided but not defined: -disk-export
```

Was an alternative feature added to replace this?

**Comment by ehsandeep** [comment, marked as answer]
@PinkDev1 https://github.com/projectdiscovery/nuclei/releases/tag/v2.3.8

Updated flag `disk-export/de` to `markdown-export/me` to keep it uniform #757
